fixtures/datagen/README.md

```diff
@@ -0,0 +1,6 @@
+## README
```

Comment:
Thank you for adding the README! I know our future selves will be happy about this in a few months.